migrations/Version202606081430001101_taoQtiItem.php

```
@@ -0,0 +1,39 @@
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addReport(
+            $this->propagate(
+                new RegisterNpmPaths()
+            )([])
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)